hello-spring/src/test/java/org/folio/hello/tests/RollTest.java

```diff
@@ -0,0 +1,46 @@
+package org.folio.hello.tests;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.lessThan;
+import static org.hamcrest.Matchers.everyItem;
+
+
+import java.util.Random;
+import java.util.Arrays;
+
+import io.restassured.response.Response;
+
+import org.folio.hello.BaseIntegrationTest;
+import org.junit.jupiter.api.Test;
+
+public class RollTest extends BaseIntegrationTest {
+  
+  @Test
+  public void testRollGet() {
+    Response response = ra()
+      .get(getRequestUrl("/roll"));
+    response.then().statusCode(200);
+
+    Integer rollResult = Integer.parseInt(response.getBody().asString());
+    assertThat(rollResult, is(both(greaterThan(0)).and(lessThan(7))));
+  }
+
+  @Test
+  public void testRollPost() {
+    Random random = new Random();
```

Comment:
SonarLint should've given you a warning in your IDE:
![image](https://user-images.githubusercontent.com/8005215/211680598-2beab6e9-d3a6-49db-ac9b-12f96318beae.png)

We generally don't like to use random values in tests, since then randomness can cause test pass or failure which is a recipe for painful debugging.

I'd recommend just using a constant value (maybe one or two different ones in different methods) to test this endpoint.

Reply:
Huh for some reasons sonarlint stopped working in my VSCode (it was working before so I never noticed). I'll take a look and fix accordingly.

Reply:
Interesting - does it say anything in the problems tab? And did you copy in the sonarlint rules list from our vscode config?

Reply:
It doesn't say anything. I'll take a look at the config file when I get back from dinner. Maybe I missed something.

Reply:
Oh yeah I got it now. Since I'm using a new computer, I haven't copied the rules list on the current computer yet. It is working now.